alf/networks/network.py

```diff
@@ -291,3 +292,40 @@ def forward(self, inputs, state=()):
     @property
     def state_spec(self):
         return self._state_spec
+
+
+@gin.configurable
+class SequentialNetwork(Network):
+    """Network composed of a sequence of torch layers."""
+
+    def __init__(self, input_tensor_spec, layers, name="SequentialNetwork"):
+        """
+
+        Args:
+            input_tensor_spec (TensorSpec): the tensor spec of the input.
+            layers (list[nn.Module]): list of torch layers.
+            name (str):
+        """
+        super().__init__(input_tensor_spec, name)
+        if len(layers) == 1 and isinstance(layers[0], nn.Sequential):
+            self._layers = layers[0]
+        else:
+            self._layers = nn.Sequential(*layers)
+
+    def forward(self, input, state=()):
+        return self._layers(input), state
+
+    def copy(self, name=None):
```

Comment:
Maybe can add a description to the ```copy``` function, which is shallow as well, similar to Network.copy.